google_gke/variables.tf

```diff
@@ -25,7 +25,6 @@ variable "name" {
 }
 
 variable "project_id" {
-  default     = null
```

Comment:
If we still want to be able to use the provider's `project_id` (accomplished with the default being `null`) we would need to go through the other areas of this module to replace any `local.project_id` references interpolated as part of a string and use something else. For now, I propose we just require this to be set.